graylog2-server/src/main/java/org/graylog2/alerts/types/DummyAlertCondition.java

```diff
@@ -42,4 +44,10 @@ public String getDescription() {
     public CheckResult runCheck() {
         return new CheckResult(true, this, this.description, Tools.iso8601(), null);
     }
+
+    @Override
+    public List<Message> getSearchHits() {
+        return null;
```

Comment:
This should return `Collections#emptyList()` instead of `null` to prevent accidental NullPointerExceptions in the caller.